package.json

```diff
@@ -17,6 +17,7 @@
         "acorn": "8.0.1"
     },
     "dependencies": {
+        "@greymass/account-creation": "^0.0.3",
```

Comment:
Is this account-creation-lib republished under a new name?

Reply:
Yeah, I guess we should either rename the npm or github package to stay consistent. Any preference?

Reply:
Just making sure it wasn't a different lib, this is fine